website/docs/usage/embeddings-transformers.mdx

```diff
@@ -745,14 +745,17 @@ To benefit from pretraining, your training step needs to know to initialize its
 this by setting `initialize.init_tok2vec` to the filename of the `.bin` file
 that you want to use from pretraining.
 
-A pretraining step that runs for 5 epochs with an output path of `pretrain/`, as
-an example, produces `pretrain/model0.bin` through `pretrain/model4.bin`. To
-make use of the final output, you could fill in this value in your config file:
```

Comment:
The default still produces all the intermediate files, right? I think I'd leave the first sentence in as something like:

> A pretraining step that runs for 5 epochs with an output path of `pretrain/`, as an example, produces `pretrain/model0.bin` through `pretrain/model4.bin` plus a copy of the final output as `pretrain/model-last.bin`.